source/extensions/filters/common/lua/lua.h

```diff
@@ -368,6 +368,21 @@ class ThreadLocalState : Logger::Loggable<Logger::Id::lua> {
         [this]() { T::registerType(tls_slot_->getTyped<LuaThreadLocal>().state_.get()); });
   }
 
+  /**
+   * Return the number of bytes used by the runtime.
```

Comment:
nit: can we specify the unit here? Based on code looks like it is in Mbs.

Reply:
It's bytes...

Reply:
oh..sorry..my bad..you are multiplying by 1024 some how I read it as you are dividing by 1024.. :-)